### DISCUSSION TITLE
Android install older versions

### DISCUSSION THREAD

**Discussion by ramondevic** [devemux86/cruiser#526] [Q&A] [answered]
How can I recover a beta version before 4.0.0?
I have 3.0.38 Jan 2024 but I think I should have more recent ones

How is it installed?
Directly running the apk will be correctly installed?

How can Cruiser be disabled from updating automatically?

Thanks.

**Comment by devemux86** [comment, marked as answer]
Android does not allow to install / overwrite an app with an older release. It requires a clean install:
Export / backup favorites / routes / profiles, uninstall, install another version, import / restore backup.

To disable the automatic updates of apps, there is an option in the settings of Google Play.

**Reply by devemux86** [reply]
Anyway why making so extraordinary moves just for 1 menu?

If changes are needed, we can discuss and implement them.

**Comment by umpfmpf** [comment]
For such cases and for paranoid users, would it be a way to add the option to back up all private settings and data at once?
Maybe even in a folder or as a complete backup in a ZIP file? And then, when playing back the backup, select which data should be imported.
And this file could also be copied to a 2. Device to synchronize the data and settings.

**Reply by devemux86** [reply]
@umpfmpf thanks, this is an interesting idea. We can think about this.

**Reply by devemux86** [reply]
Please create a new topic for this idea so it doesn't get forgotten here.
